### ISSUE TITLE
problem with peaks that are viewpoints

### ISSUE THREAD

**Issue by mkoniecz** [gravitystorm/openstreetmap-carto#1372]
Peaks are frequently also great viewpoints. On printed map it is typically solved either by special icon for such situation or by shifting where viewpoint is marked (typically in direction of the most impressive view).

We are not doing the first one and second is tagging for renderer.

Before #1349 [natural=peak; tourism=viewpoint] was displayed as peak, now it is displayed as viewpoint - https://cloud.githubusercontent.com/assets/899988/6650865/ade346d2-ca27-11e4-8bd6-b58cefe5462b.png.

I propose to display it as special icon that would be merge of peak and viewpoint icons. @nebulon42 - can you maybe produce such icon?

**Comment by nebulon42** [comment]
Yes, I can produce such an icon as viewpoint and peak is on my list anyway. But first I would like to wait for some consensus on whether a special icon is required/wanted or not.

**Comment by brycenesbitt** [comment]
I think a peak icon with a ring of viewpoint icon would look great.
Better yet if the direction of view were properly taggable.

**Comment by Tomasz-W** [comment]
Peak and viewpoint icons merged:

![natural peak tourism viewpoint](https://user-images.githubusercontent.com/25656654/36922089-0e1d0704-1e67-11e8-953d-bc05d1530566.png)

**Comment by Tomasz-W** [comment]
@matkoniecz @nebulon42 @kocio-pl 
Are you satysfied by icon above, or you want something different?

**Comment by kocio-pl** [comment]
First of all I'm not sure if that's good idea, but I would have to see the rendering to tell how it works visually. For example: which color should it be?

**Comment by matkoniecz** [comment]
> "By definition", a peak is a viewpoint unless it's a smoothed hilltop covered by forest.

Nearly all peaks covered by forest are not viewpoints, many small peaks are not viewpoints, sometimes viewpoint is not at this highest point (peak), sometimes peak is not considered as viewpoint as view is blocked...

**Comment by jeisenbe** [comment]
@Tomasz-W - is there an SVG icon available to try a test rendering? Or could you make a test by just copy-and-pasting the icon over a screenshot?

**Comment by ppete2** [comment]
I'm not quite sure wheter it's a good idea or not to render them with a combined icon. Situation right now is, that peak icon+name+elevation is rendered.

I know many peaks with scenic views - which are not extra tagged as viewpoint, although they would "deserve" it. I think many mappers think, that a lot of peaks "inherently" provide scenic views (besides from peaks surrounded by wood) - no need to extra tag them as viewpoint too. So they just use `tourism=viewpoint` for objects/locations which are not peaks, eg locations where a panorama board or binoculars are placed, or an observation tower.

All in all I'm not sure if adding a "viewpoint"-icon to peaks would add much of important info to a map. And it could lead mappers to start adding all non-wooded peaks with `toursim=viewpoint`, cause otherwise they look "less important" on a map than peaks already having this tag. Which in my mind would not a good developement regarding peaks.

**Comment by imagico** [comment]
To be clear - the idea of using a combined symbol for features tagged both `natural=peak` and `tourism=viewpoint` is still seen positively in principle.  But you have to keep in mind that often the peak and the viewpoint are not exactly the same position and hence both are mapped separately.  This is probably a more common situation than the double tagged node (currently 2271 uses).  Since close-by nodes will block each other in terms of symbols, having a combined symbol for double tagged nodes would incentivize this mapping in favor of the separate nodes.

Aggregating close-by nodes with a combined symbols would be possible (and interesting not only for peak/viewpoint but also for other combinations) but the limitations of mapnik make practical implementation of this without the risk of duplicate symbolization difficult.
